src/main/java/baseball/Application.java

```diff
@@ -1,7 +1,23 @@
 package baseball;
 
+import java.util.List;
+
 public class Application {
+
     public static void main(String[] args) {
-        //TODO: 숫자 야구 게임 구현
+        Game game = new Game();
+        Util util = new Util();
+        List<Integer> computer = util.initComputer();
+        while (true) {
+            List<Integer> input = util.input();
+            String result = game.play(input, computer);
+            if (result.equals("1")) {
+                computer = util.initComputer();
+            } else if (result.equals("2")) {
+                return;
+            } else if (!result.equals("")) {
```

Comment:
```suggestion
            } else if (!result.isEmpty()) {
```

비어있는것을 확인할때는 isEmpty()를 사용하시는 것이 가독성이 더욱 좋습니다 :)

Reply:
좋은 정보 감사합니다!! 

비어있는 문자를 확인할 땐 isEmpty()를 이용하겠습니다!